### PR TITLE
BUG: Fix bug with orphan sg_api_usage

### PR DESCRIPTION
**Pull request by larsoner**
Fixes this bug when running in noplot mode:
```
checking consistency... /Users/larsoner/python/mne-python/doc/sg_api_usage.rst: WARNING: document isn't included in any toctree
```
The `:orphan:` and title should always be added.

Also snuck in some `changelog_generator` stuff to speed things up that I've been using elsewhere that should help at release time.

### CODE REVIEW

File: .github_changelog_generator [hunk 1]
```diff
@@ -1,5 +1,7 @@
 project=sphinx-gallery
 user=sphinx-gallery
 add-sections={"api":{"prefix":"**API changes**","labels":["api"]},"documentation":{"prefix":"**Documentation**","labels":["documentation"]},"maintenance":{"prefix": "**Project maintenance**","labels":["maintenance"]}}
-max-issues=200
+max-issues=2
```
Comment (lucyleeow) [inline .github_changelog_generator:4]: Why do we only want to fetch 2 issues? We should want a few months worth right?

**Review by lucyleeow — APPROVED** [review]
LGTM, thanks! Just one question.

File: .github_changelog_generator (hunk 1 above)
Comment (larsoner) [reply .github_changelog_generator:4]: We actually already have `issues=no` meaning "don't put any issues . It's a mystery to me why it even bothers fetching issues when this is the case but it did (at least at some point).

Comment (lucyleeow) [reply .github_changelog_generator:4]: Ah right, did not realise. So for the changelog we only use PRs? I thought we, at some point, used issues as well?

Comment (larsoner) [reply .github_changelog_generator:4]: Yes we used to but in practice they didn't seem to add much value compared to properly listing PRs so I removed them

Comment (lucyleeow) [reply .github_changelog_generator:4]: Good to know, I tend to agree. Some issues are just quickly resolved without anything needing to be done and thus did not really add anything to the changelog.

**Pull request merged**